lib/src/audio_waveforms.dart

```diff
@@ -41,7 +41,7 @@ class _AudioWaveformsState extends State<AudioWaveforms> {
   Offset _dragOffset = Offset.zero;
 
   double _initialOffsetPosition = 0.0;
-  double _initialPosition = 0.0;
+  double _initialPosition = 3.0;
```

Comment:
Dont use static values. What if I set waveThickness too large wouldn't the wave get outside of the viewport?